### ISSUE TITLE
rename releases to support GOOS returned values ie darwin, linux

### ISSUE THREAD

**Issue by 2ffs2nns** [aquasecurity/trivy#446]
I'd love to use some automation to download a given release `.tar.gz` file, and base it on the calling clients operating system. I use a golang client and would want/expect the filename to match one of the supported values. https://golang.org/doc/install/source#environment

```Choices for $GOOS are android, darwin (macOS 10.11 and above and iOS), dragonfly, freebsd, illumos, js, linux, netbsd, openbsd, plan9, solaris and windows.```

In my case, that's either `darwin` or `linux`  not `macOS` or `Linux`

The list of releases/tar files I'm looking at end up here https://github.com/aquasecurity/trivy/releases

**Comment by simar7** [comment]
Thanks for reporting this in @2ffs2nns. We'll have a look into it.

cc @knqyf263

**Comment by knqyf263** [comment]
Fair enough. You mean trivy_0.6.0_darwin_64bit.tar.gz instead of trivy_0.6.0_macOS-64bit.tar.gz, right?

**Comment by 2ffs2nns** [comment]
@knqyf263 yup, exactly `trivy_0.6.0_darwin_64bit.tar.gz` is what I'm looking for

**Comment by knqyf263** [comment]
I see, but we have to be careful because the change fails a download with current name and affects a lot of users.
